.gitignore

```diff
@@ -34,3 +34,6 @@ Thumbs.db
 
 # Files that can be automatically downloaded that we don't want to ship with our builds
 /src/extensibility/node/node_modules/request/tests/
+
+# .disabled should be ignored, it could be used to disable default extensions
+.disabled
```

Comment:
@MarcelGerber, actually I misunderstood you, you were saying the rule itself should be more specific and I understood you were referring to the comment text. Will fix this.